sysutils/tiptop/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github  1.0
+PortGroup           python  1.0
+
+github.setup        nschloe tiptop 0.0.16 v
+github.tarball_from archive
+revision            0
+
+description         \
+    Command-line system monitoring
+
+long_description    \
+    tiptop is a command-line system monitoring tool in the spirit of top. It \
+    displays various interesting system stats, graphs it, and works on Linux \
+    and macOS.
+
+platforms           darwin
+license             MIT
+categories          sysutils
+maintainers         {gmail.com:herby.gillot @herbygillot} openmaintainer
+
+checksums           rmd160  a4ae261c51af8ebb5b736c6c2cee39a8658837ad \
+                    sha256  73f8a4367f05acc90a118554bfd5bfd9b645a30b1de4985740b1cf6a1a6486c2 \
+                    size    14932
+
+python.default_version      39
+python.pep517               yes
+
+depends_build-append    \
+                    port:py${python.version}-setuptools \
+                    port:py${python.version}-wheel
+
+depends_run-append  port:py${python.version}-cpuinfo \
+                    port:py${python.version}-distro \
+                    port:py${python.version}-importlib-metadata \
+                    port:py${python.version}-psutil \
+                    port:py${python.version}-textual
+
+depends_test-append port:py${python.version}-pytest
+
+test.run            yes
+test.env-append     PYTHONPATH=./build/lib
+test.cmd            pytest-${python.branch}
+test.pre_args       {}
```

Comment:
why is this line needed - typically one sets `test.target` to an empty string. I've never needed to clear `test.pre_arg` before...

Reply:
`test.pre_args` is defaulting to the value `test`:
https://github.com/macports/macports-base/blob/999867a094363e006d14bd714be9558933b526be/src/port1.0/porttest.tcl#L23-L24

If not cleared, this happens:
```
Executing:  cd "/opt/macports-test/var/macports/build/_Users_herby_Source_macports-ports_sysutils_tiptop/tiptop/work/tiptop-0.0.16" && pytest-3.9 test
DEBUG: system:  cd "/opt/macports-test/var/macports/build/_Users_herby_Source_macports-ports_sysutils_tiptop/tiptop/work/tiptop-0.0.16" && pytest-3.9 test
ERROR: file or directory not found: test

================================================================================================================ test session starts ================================================================================================================
platform darwin -- Python 3.9.9, pytest-6.2.5, py-1.10.0, pluggy-1.0.0
rootdir: /opt/macports-test/var/macports/build/_Users_herby_Source_macports-ports_sysutils_tiptop/tiptop/work/tiptop-0.0.16
collected 0 items

=============================================================================================================== no tests ran in 0.00s ===============================================================================================================
Command failed:  cd "/opt/macports-test/var/macports/build/_Users_herby_Source_macports-ports_sysutils_tiptop/tiptop/work/tiptop-0.0.16" && pytest-3.9 test
Exit code: 4
```

Reply:
okay, so then please do the same as in pretty much every other Python port (and I suggested above) and add the line
```test.target```

Reply:
Changes made.